src/pages/hez-circ-supply.html

```diff
@@ -31,4 +31,4 @@
 </head>
 <body>
   <pre class="total-hez">4700000</pre>
```

Comment:
This static page should be now created as a route in the Gatsby app.

Reply:
Added as a page.